File: patch_app_module.js
const fs = require('fs');
let content = fs.readFileSync('backend/src/app.module.ts', 'utf8');
content = "import { MrpModule } from './mrp/mrp.module';\n" + content;
content = content.replace("ShopeeModule, AiModule, ManufacturingModule", "ShopeeModule, AiModule, ManufacturingModule, MrpModule");
fs.writeFileSync('backend/src/app.module.ts', content, 'utf8');

File: patch_app_module.py
with open("backend/src/app.module.ts", "r", encoding="utf-8") as f:
    c = f.read()

if "import { ApprovalModule }" not in c:
    c = "import { ApprovalModule } from './approval/approval.module';\n" + c

with open("backend/src/app.module.ts", "w", encoding="utf-8") as f:
    f.write(c)

